src/Console/InputDefinition.php

```diff
@@ -23,6 +23,7 @@ public function __construct()
 
         $this->addOption(new InputOption('grep', 'g', InputOption::VALUE_REQUIRED, 'Run tests matching <pattern> <comment>(default: *.spec.php)</comment>'));
         $this->addOption(new InputOption('no-colors', 'C', InputOption::VALUE_NONE, 'Disable output colors'));
+        $this->addOption(new InputOption('--force-colors', '-f', InputOption::VALUE_NONE, 'Force output colors'));
```

Comment:
I am wondering if we should just omit the short switch here? Maybe its ok and more explicit for this option to always require the long form? `peridot --force-colors specs`

Thoughts?